lib/WebService/Stripe.pm

```diff
@@ -185,6 +191,14 @@ method get_bitcoin_receiver(Str $id, :$headers) {
     return $self->get( "/v1/bitcoin/receivers/$id", {}, headers => $headers );
 }
 
+method _stripe_opts_to_headers(HashRef $opts) {
+    my $headers = $opts->{'headers'} // {};
+    if ($opts->{'stripe_account'}) {
+        $headers->{'Stripe-Account'} = $opts->{'stripe_account'};
+    }
+    return $headers;
+}
```

Comment:
Would be nice if `WebService::Client` exposed the underlying HTTP::Request object. I think we could build cleaner interfaces faster. E.g. syntax:

```perl
my $request = $self->build_request($url, $data, $headers);
$self->apply_options($request, $options);
return $self->submit($request);
```